tide_grant.module

```diff
@@ -145,3 +146,29 @@ function tide_grant_migration_plugins_alter(array &$migrations) {
     $migrations['grants_feed_importer']['source']['urls'] = $url;
   }
 }
+
+/**
+ * Implements hook_node_insert().
+ */
+function tide_grant_node_insert(NodeInterface $node) {
+  // Everytime the feed will import contents, this will trigger.
+  // Only update the grant nodes created by anonymous user.
+  if (!empty($node) && $node->bundle() == 'grant' && $node->getOwnerId() == 0) {
```

Comment:
It's not a bad check to have but I think because `$node` is typed and doesn't allow `NULL` you can skip this check.